packages/vue-test-workspace-vue-2/vue-tsc/strict-template/intrinsicProps/main.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+	<div>
+		<!-- FIXME: bug here -->
+		<!-- @vue-expect-error -->
```

Comment:
Seems no longer an issue here with the latest vue version? I got an "error TS2578: Unused '@ts-expect-error' directive" in the ecosystem-ci
https://github.com/vuejs/ecosystem-ci/actions/runs/6608899447/job/17948286761#step:7:7158

Reply:
Never mind. It's fixed by https://github.com/vuejs/language-tools/pull/3683